api/src/main/resources/workbench-api.yaml

```diff
@@ -3831,7 +3813,7 @@ paths:
           description: information about counts for EHR domains, physical Measurement CSS, and Surveys for all survey names
           schema:
             "$ref": "#/definitions/CardCountResponse"
-  "/v1/cohortbuilder/{workspaceNamespace}/{workspaceId}/criteria/{domain}/{standard}/search/term":
```

Comment:
The domain and standard fields got moved into the body of this request and is no longer needed in path.